src/core/exec-invoke.c

```diff
@@ -1,5 +1,6 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <linux/sched.h>
```

Comment:
please keep includes sorted (unless there's a strong reason not to sort them, kernel headers are pretty poorly put together usually that sometimes order mattaers)

Reply:
Hmm? It's already sorted?

Reply:
Do you maybe mean to place the `linux/` include after the `sys/` ones?